code/datums/datacore.dm

```diff
@@ -315,16 +313,7 @@ GLOBAL_DATUM_INIT(data_core, /datum/datacore, new)
 		locked += L
 	return
 
-/datum/datacore/proc/get_id_photo(mob/living/carbon/human/human, client/client, show_directions = list(SOUTH))
-	var/datum/job/humans_job = human.mind.assigned_role
-	var/datum/preferences/humans_prefs
-	if(!client)
-		client = human.client
-	if(client)
-		humans_prefs = client.prefs
-	if (human.dna.species.roundstart_changed)
-		return get_flat_human_icon(null, humans_job, null, DUMMY_HUMAN_SLOT_MANIFEST, show_directions)
-	else
-		return get_flat_human_icon(null, humans_job, humans_prefs, DUMMY_HUMAN_SLOT_MANIFEST, show_directions)
+/datum/datacore/proc/get_id_photo(mob/living/carbon/human/human, show_directions = list(SOUTH))
+	return get_flat_existing_human_icon(human, show_directions)
 
 #undef DUMMY_HUMAN_SLOT_MANIFEST
```

Comment:
can you remove this define? It was only used here.

Reply:
I can, yeah. Give me a moment and I'll push it.